### PR TITLE
Add green "available on roadie" tags

### PR DESCRIPTION
**Pull request by dtuite**
The idea is to try to help Roadie customers who accidentally going to the wrong plugin directory.

Preview link: https://deploy-preview-897--roadie.netlify.app/backstage/plugins/argo-cd/

Screenshot 2022-11-29 at 20 55 26

### CODE REVIEW

File: src/templates/Plugin.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const PluginTemplate = ({ data }) => {
           {plugin.frontmatter.gettingStarted && (
             <ResponsiveSpacer>
               <div className="text-center pb-3" >
-                <Title text="Installation steps" />
+                <Title text="Self-hosted Backstage installation steps" />
```
Comment (dtuite) [inline src/templates/Plugin.js:69]: I'm trying to make it more clear that these are instructions for self-hosted Backstage. Not for Roadie users.

File: src/templates/Plugin.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ const PluginTemplate = ({ data }) => {
 
           <ResponsiveSpacer>
             <div className="text-center pb-3">
-              <Title text="Getting started is simple" />
+              <Title text="Get SaaS Backstage" />
```
Comment (dtuite) [inline src/templates/Plugin.js:55]: Stronger selling language.

File: content/backstage/plugins/tech-insights.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ logoImage: '../../assets/tech-insights/tech_insights.png'
 coverImage: '../../assets/tech-insights/tech_insights_scorecard.png'
 coverImageAlt: "Visualize, understand and optimize your team's tech health."
 
-
+availableOnRoadie: true
```
Comment (dtuite) [inline content/backstage/plugins/tech-insights.md:19]: In some cases there is no docs for a plugin in the Roadie docs.

**Review by jorgelainfiesta — APPROVED** [review]
Looks good! a next step could be to show a tabbed menu on the instructions, the currently selected is "Self-hosted," and the other tab is "Roadie". When a user clicks on "Roadie" tab they are taken to the docs page.

**Pull request merged**